soc/arm/st_stm32/stm32h7/power_m7.c

```diff
@@ -48,8 +48,15 @@ static int stm32_power_init(const struct device *dev)
 	LL_PWR_ConfigSupply(LL_PWR_LDO_SUPPLY);
 #endif
 	LL_PWR_SetRegulVoltageScaling(LL_PWR_REGU_VOLTAGE_SCALE1);
+
+	/*
+	 * At this point it should actually only be necessary to wait for ACTVOSRDY during startup.
+	 * Anyway, whenever the voltage scaling is changed one you should also wait for VOSRDY.
```

Comment:
Comment is a bit confusing even if I finally got it in the end.
Could you have a true to make it clearer ?

Reply:
I agree, it is confusing. Hopefully(?) done.

Reply:
Adapted the comment slightly.